scripts/run_integration_tests_windows.sh

```diff
@@ -17,6 +17,13 @@ TARGET_CC="musl-gcc"
 CFLAGS="-I /usr/include/x86_64-linux-musl/ -idirafter /usr/include/"
 fi
 
+# Use device mapper to create a snapshot of the Windows image
+loop_device=$(losetup --find --show --read-only /root/workloads/windows-server-2019.raw)
+dmsetup create windows-base --table "0 41943040 linear $loop_device 0"
```

Comment:
Can you get this from the file rather than hardcoding it: `du -b -B 512 ...`

Reply:
Yes definitely. I wanted to validate that everything was working as expected.